src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -512,24 +531,26 @@ export const VAFileNumberDescription = (
 );
 
 const PhoneViewField = ({ formData: phoneNumber, name }) => {
-  const isDomestic = phoneNumber.length <= 10;
-  const midBreakpoint = isDomestic ? -7 : -8;
-  const lastPhoneString = `${phoneNumber.slice(-4)}`;
-  const middlePhoneString = `${phoneNumber.slice(midBreakpoint, -4)}-`;
-  const firstPhoneString = `${phoneNumber.slice(0, midBreakpoint)}-`;
+  const safePhone = phoneNumber || '';
+  const midBreakpoint = -7;
+  const lastPhoneString = `${safePhone.slice(-4)}`;
+  const middlePhoneString = `${safePhone.slice(midBreakpoint, -4)}-`;
+  const firstPhoneString = `${safePhone.slice(0, midBreakpoint)}-`;
 
   const phoneString = `${firstPhoneString}${middlePhoneString}${lastPhoneString}`;
```

Comment:
So I know you didn't do this, but can you move those dashes down to this line, please? This confused me for a second why we needed to break this up only to slap it all together again. As a bonus, we also won't need all those string templates above.

Reply:
👍